Pod/Classes/SKYContainer.m

```diff
@@ -99,7 +98,7 @@ - (SKYPublicDatabase *)publicCloudDatabase
 - (void)configAddress:(NSString *)address
```

Comment:
I think `configureAddress:` and `configureWithAPIKey:` should be marked as deprecated.